test/parallel/test-shadow-realm-preload-module.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const common = require('../common');
+const fixtures = require('../common/fixtures');
+const assert = require('assert');
+
+const commonArgs = [
+  '--experimental-shadow-realm',
+];
+
+async function main() {
+  // Verifies that --require preload modules are not enabled in the ShadowRealm.
+  const child = await common.spawnPromisified(process.execPath, [
```

Comment:
Can you use `spawnSyncAndExitWithoutError` from `test/common/child_process` here instead? That would dump more information to the logs if the process doesn't exit in the way expected for easier debugging for the CI.